sql/2024/privacy/ccpa_most_common_phrases.sql

```diff
@@ -0,0 +1,5 @@
+WITH pages_with_phrase AS (
+  SELECT client, rank_grouping, page, count(DISTINCT page) OVER (PARTITION BY client, rank_grouping) AS total_pages_with_phrase_in_rank_group, JSON_QUERY_ARRAY(custom_metrics, '$.privacy.ccpa_link.CCPALinkPhrases') AS ccpa_link_phrases FROM `httparchive.all.pages`, --TABLESAMPLE SYSTEM (0.01 PERCENT)
+    UNNEST([1000, 10000, 100000, 1000000, 10000000, 100000000]) AS rank_grouping WHERE date = '2024-06-01' AND is_root_page = true AND rank <= rank_grouping AND array_length(JSON_QUERY_ARRAY(custom_metrics, '$.privacy.ccpa_link.CCPALinkPhrases')) > 0
+)
+SELECT client, rank_grouping, link_phrase, count(DISTINCT page) AS num_pages, count(DISTINCT page) / any_value(total_pages_with_phrase_in_rank_group) AS pct_pages FROM pages_with_phrase, unnest(ccpa_link_phrases) link_phrase GROUP BY link_phrase, rank_grouping, client ORDER BY rank_grouping, client, num_pages DESC
```

Comment:
Queries like this are very long and difficult to parse. Can we format them to our prefered style (unfortunately the linter doesn't insert newlines):

```sql
```suggestion
WITH pages_with_phrase AS (
  SELECT
    client,
    rank_grouping,
    page,
    COUNT(DISTINCT page) OVER (PARTITION BY client, rank_grouping) AS total_pages_with_phrase_in_rank_group,
    JSON_QUERY_ARRAY(custom_metrics, '$.privacy.ccpa_link.CCPALinkPhrases') AS ccpa_link_phrases
  FROM
    `httparchive.all.pages`, --TABLESAMPLE SYSTEM (0.01 PERCENT)
  UNNEST([1000, 10000, 100000, 1000000, 10000000, 100000000]) AS rank_grouping
  WHERE
    date = '2024-06-01' AND
    is_root_page = true AND
    rank <= rank_grouping AND
    array_length(JSON_QUERY_ARRAY(custom_metrics, '$.privacy.ccpa_link.CCPALinkPhrases')) > 0
)
SELECT
  client,
  rank_grouping,
  link_phrase,
  COUNT(DISTINCT page) AS num_pages,
  count(DISTINCT page) / ANY_VALUE(total_pages_with_phrase_in_rank_group) AS pct_pages
FROM
  pages_with_phrase,
  UNNEST(ccpa_link_phrases) link_phrase
GROUP BY
  link_phrase,
  rank_grouping,
  client
ORDER BY
  rank_grouping,
  client,
  num_pages DESC
```

Also built-in functions like `COUNT`, and `ANY_VALUE` should be in capitals.

(This applies to quite a few of these queries, not just this one)

Reply:
Oh, there are quite a few of similar ones.
Got some assistance fixing by temporarily enabling L016 (too long lines) in sqlfluff.